pycheribuild/projects/run_qemu.py

```diff
@@ -28,6 +28,9 @@
 # SUCH DAMAGE.
 #
 import socket
+from random import randint
+
+import libtmux
```

Comment:
I try hard to make cheribuild work with only the python standard library installed (that's why I bundle dependencies in the 3rdparty directory). This should be
```python
try:
     import libtmux
 except ImportError:
    .... fall back to something else
```

Also it shouldn't be necessary unless you have `self.config.debug_kernel` set. If it's only used within one function, I'd import it there to reduce startup time (important for bash tab-completion speed).